src/js/post-911-gib-status/actions/post-911-gib-status.js

```diff
@@ -1,18 +1,63 @@
-import { apiRequest } from '../utils/helpers';
+import Raven from 'raven-js';
+
+import { apiRequest } from '../../common/helpers/api';
+
+import {
+  BACKEND_AUTHENTICATION_ERROR,
+  BACKEND_SERVICE_ERROR,
+  GET_ENROLLMENT_DATA_FAILURE,
+  GET_ENROLLMENT_DATA_SUCCESS,
+  NO_CHAPTER33_RECORD_AVAILABLE,
+  metaStatus
+} from '../utils/constants';
 
 export function getEnrollmentData() {
   return (dispatch) => {
-    apiRequest('/v0/post911_gi_bill_status',
+    apiRequest(
+      '/post911_gi_bill_status',
       null,
       (response) => {
+        // TODO: remove this once vets-api stops returning 200s that pass
+        // errors in the `meta` object. Instead, it will transform these into
+        // non-200 status codes so the error callback will handle them.
+        if (response.meta) {
+          if (response.meta.status === metaStatus.SERVER_ERROR ||
+              response.meta.status === metaStatus.NOT_FOUND) {
+            return dispatch({ type: BACKEND_SERVICE_ERROR });
+          }
+          if (response.meta.status === metaStatus.NOT_AUTHORIZED) {
+            return dispatch({ type: BACKEND_AUTHENTICATION_ERROR });
+          }
+        }
         return dispatch({
-          type: 'GET_ENROLLMENT_DATA_SUCCESS',
+          type: GET_ENROLLMENT_DATA_SUCCESS,
           data: response.data.attributes,
         });
       },
-      () => dispatch({
-        type: 'GET_ENROLLMENT_DATA_FAILURE'
+      (response) => {
+        if (response.status === 503 || response.status === 504) {
+          // Either EVSS or a partner service is down or EVSS times out
+          return dispatch({ type: BACKEND_SERVICE_ERROR });
+        }
+        if (response.status === 403) {
+          // Backend authentication problem
+          return dispatch({ type: BACKEND_AUTHENTICATION_ERROR });
+        }
+        if (response.status === 404) {
+          // EVSS partner service has no record of this user
+          return dispatch({ type: NO_CHAPTER33_RECORD_AVAILABLE });
+        }
+        throw (response);
```

Comment:
It's actually to trigger the `catch` below. So if none of the response status codes are things we recognize, then something weird and unexpected is happening, so the below `catch` will log a sentry error. 

This one I did test by hacking the vets-api controller to raise an unexpected error.